package.json

```diff
@@ -101,7 +101,7 @@
     "xml": "1.0.1"
   },
   "optionalDependencies": {
-    "sharp": "0.20.7",
+    "sharp": "0.21.0",
```

Comment:
The reason for bump - http://sharp.pixelplumbing.com/en/stable/changelog/#v0210-4th-october-2018, `size` attribute has been introduced in this version which makes image size calculation easier.

Reply:
> Requires libvips v8.7.0.

What does that mean exactly?
Does it mean 0.21.0 no longer works with < 8.7.0?

Reply:
My understanding it's just version of binaries that are pinned to a specific release, basing on [this](http://sharp.pixelplumbing.com/en/stable/install/#linux):
> libvips and its dependencies are fetched and stored within node_modules/sharp/vendor during npm install. This involves an automated HTTPS download of approximately 8MB.

and how version installation is handled around [here](https://github.com/lovell/sharp/blob/master/install/libvips.js#L75)

Reply:
Thanks. It probably only breaks if 

>  const useGlobalLibvips = libvips.useGlobalLibvips();

Is true. That means you are using a global libvips installation? And you dont have 8.7.0 installed.